libs/features/mas/src/external.js

```diff
@@ -2,7 +2,6 @@
 import {
     CheckoutType,
     WorkflowStep,
-    buildCheckoutUrl,
 } from '@pandora/commerce-checkout-url-builder';
```

Comment:
lets remove the whole `'@pandora/commerce-checkout-url-builder'` dependency and the .tgz package from internal folder?